src/pages/ServiceMessages.tsx

```diff
@@ -2,41 +2,36 @@ import { isEmpty, map } from "lodash-es"
 
 import { useServiceMessages } from "../api/requests"
 import ServiceMessage from "../components/ServiceMessage"
-import { Header1 } from "../components/text"
 import { cl } from "../utils"
 
-import Page from "./Page"
-
+import BasicPage from "./BasicPage"
 
 const ServiceMessages = () => {
   const { data: serviceMessages, isLoading } = useServiceMessages()
 
   if (isLoading) return null
 
   return (
-    <Page className="py-24 px-16 md:px-24 mx-8 md:mx-16 bg-purple-700 rounded-md space-y-16">
-      <div className="text-center">
-        <Header1>Driftsmeldinger</Header1>
-      </div>
-      <div className="grid grid-cols-1 gap-8 justify-items-center">
-        {isEmpty(serviceMessages)
-          ? <div>🎄 Ingen driftsmeldinger. Livet er herlig! 🎄</div>
-          : map(serviceMessages, (serviceMessage) => {
-            const messageClasses = serviceMessage.resolved
-              ? "border-yellow-400/70 text-gray/70"
-              : "border-red-700/70"
-
-            return (
-              <ServiceMessage
-                key={serviceMessage.uuid}
-                className={cl("border-2 p-8 rounded-md w-full max-w-320", messageClasses)}
-                serviceMessage={serviceMessage}
-              />
-            )
-          })
-        }
-      </div>
-    </Page>
+    <BasicPage
+      title="Driftsmeldinger"
+      className="flex flex-col items-center flex-1"
```

Comment:
Dette styler basically bare headeren, right? Trengs `flex-col` og `flex-1`? Trengs noe i det hele tatt egentlig?

Om du mente å style innholdet så er det `containerClassName`. Kanskje forvirrende, åpent for bedre navn der... Anyway, den er grid rows by default så er vel unødvendig å overskrive med flex om ikke du skal flexe noe veldig spesifikt.